### ISSUE TITLE
Cannot Provide Pagination Parameters to .listSubmissions()

### ISSUE THREAD

**Issue by tkcranny** [webflow/js-webflow-api#155]
```typescript
const webflow = new WebflowClient({ accessToken: ... });

await webflow.forms.listSubmissions(formId); // Cannot paginate
```

Pretty much this. The [API returns Pagination data](https://developers.webflow.com/data/reference/list-submissions), but there are no parameters provided to control which page is desired. This is presumably a limitation of the RESTful API and this generated SDK both.

**Comment by zplata** [comment]
Thanks @tkcranny - we've identified the issue and will get this resolved in the next SDK release (and the API playground in our data docs should reflect this shortly)

**Comment by zplata** [comment]
Hey @tkcranny - this should be addressed in `v2.3.6`, and you can now pass in the pagination options (i.e. `offset`, `limit`):
https://github.com/webflow/js-webflow-api/releases/tag/v2.3.6

Feel free to open a new issue if you still have troubles here, thanks!
